scripts/popup.js

```diff
@@ -23,6 +23,13 @@ function view_all_function(){
 	document.location.href = open_url;
 }
 
+//Sharing on Social Media Social 
+function share_on_socialMedia()
+{
+	srch_url = $('#search').val();
```

Comment:
You are missing a `var srch_url`.
Also please remove jQuery. It should be straightforward. Thank you.